### DISCUSSION TITLE
How to uninstall Sandboxie-Plus in portable mode?

### DISCUSSION THREAD

**Discussion by somethimg6010** [sandboxie-plus/Sandboxie#3827] [General]
Hello, I'm having trouble uninstalling the program.

**Comment by somethimg6010** [comment]
The program is in portable mode

**Reply by LumitoLuma** [reply]
There should be an option within the application:

![imagen](https://github.com/sandboxie-plus/Sandboxie/assets/50250351/e8405c92-28d1-4d70-a0a1-f6a6714e9400)

After that, you can safely remove the app folder.

**Reply by bastik-1001** [reply]
I am not using the portable version, so I am not sure if you can set the shell integration with it and if that is removed by uninstall all. If it can be set, you disable them in the Global Settings > Shell Integration.
